lib/debug/repl.cpp

```diff
@@ -609,4 +636,38 @@ void Repl::addTemperatureCommands(const std::uint8_t bus, const std::uint8_t dev
   addCommand(temperature_read_command);
 }
 
+void Repl::addLowPowerCurrentCommands(const std::uint8_t bus, const std::uint8_t device_address)
+{
+  const auto optional_i2c = io::HardwareI2c::create(logger_, bus);
+  if (!optional_i2c) {
+    logger_.log(core::LogLevel::kFatal, "Failed to create I2C instance on bus %d", bus);
+    return;
+  }
+  const auto i2c = std::move(*optional_i2c);
+  const auto optional_low_power_current
```

Comment:
You don't check this actually returns something despite using a create function?

Reply:
An issue which can and should be considered when focusing on the LPC sensor - This code shouldn't have been edited with this PR
As such, all LPC sensor debug code has now (hopefully) been removed from this PR